src/iodev.h

```diff
@@ -11,6 +11,12 @@ typedef enum _iodev_id_t {
     IODEV_FB,
     IODEV_USB0,
     IODEV_USB1,
+    IODEV_USB2,
+    IODEV_USB3,
+    IODEV_USB4,
+    IODEV_USB5,
+    IODEV_USB6,
+    IODEV_USB7,
     IODEV_USB_MAX,
```

Comment:
I'd prefer it if MAX is inclusive, i.e. equal to USB7, not to one after, given the name.

Reply:
with `IODEV_USB_VUART` moved from the end IODEV_MAX is enough